lib/gds_api/test_helpers/publishing_api_v2.rb

```diff
@@ -5,698 +5,8 @@
-      alias_method :publishing_api_has_linkables, :stub_publishing_api_has_linkables
-      alias_method :publishing_api_has_item, :stub_publishing_api_has_item
-      alias_method :publishing_api_has_item_in_sequence, :stub_publishing_api_has_item_in_sequence
-      alias_method :publishing_api_does_not_have_item, :stub_publishing_api_does_not_have_item
-      alias_method :publishing_api_has_links, :stub_publishing_api_has_links
-      alias_method :publishing_api_has_expanded_links, :stub_publishing_api_has_expanded_links
-      alias_method :publishing_api_has_links_for_content_ids, :stub_publishing_api_has_links_for_content_ids
-      alias_method :publishing_api_does_not_have_links, :stub_publishing_api_does_not_have_links
-      alias_method :publishing_api_has_lookups, :stub_publishing_api_has_lookups
-      alias_method :publishing_api_has_linked_items, :stub_publishing_api_has_linked_items
-      alias_method :publishing_api_get_editions, :stub_publishing_api_get_editions
-
-    private
-
-      def stub_publishing_api_put(*args)
-        stub_publishing_api_postlike_call(:put, *args)
-      end
-
-      def stub_publishing_api_patch(*args)
-        stub_publishing_api_postlike_call(:patch, *args)
-      end
-
-      def stub_publishing_api_postlike_call(method, content_id, body, resource_path, override_response_hash = {})
-        response_hash = { status: 200, body: "{}", headers: { "Content-Type" => "application/json; charset=utf-8" } }
-        response_hash.merge!(override_response_hash)
-        response_hash[:body] = response_hash[:body].to_json if response_hash[:body].is_a?(Hash)
-        url = PUBLISHING_API_V2_ENDPOINT + resource_path + "/" + content_id
-        stub_request(method, url).with(body: body).to_return(response_hash)
-      end
-
-      def deep_stringify_keys(hash)
-        deep_transform_keys(hash, &:to_s)
-      end
-
-      def deep_transform_keys(object, &block)
-        case object
-        when Hash
-          object.each_with_object({}) do |(key, value), result|
-            result[yield(key)] = deep_transform_keys(value, &block)
-          end
-        when Array
-          object.map { |item| deep_transform_keys(item, &block) }
-        else
-          object
-        end
-      end
-
-      def resource_not_found(content_id, type)
-        {
-          error: {
-            code: 404,
-            message: "Could not find #{type} with content_id: #{content_id}",
-          },
-        }
-      end
-
-      def version_conflict(expected_version, actual_version = expected_version + 1)
-        {
-          error: {
-            code: 409,
-            message: "A lock-version conflict occurred. The `previous_version` you've sent (#{expected_version}) is not the same as the current lock version of the edition (#{actual_version}).",
-            fields: { previous_version: ["does not match"] },
-          },
-        }
-      end
+      include PublishingApi
```

Comment:
We've got a deprecation warning opportunity here: https://github.com/alphagov/gds-api-adapters/blob/5d732cc35ffc7441da6c7cfb7ac3fdb5d56464dc/lib/gds_api/test_helpers/rummager.rb#L10-L12